pkg/zonemgr/zone_file_cache_test.go

```diff
@@ -111,7 +117,7 @@ var _ = Describe("cached zone file content maintenance", func() {
 
 		When("interfaces records list contains single vmi", func() {
 			BeforeEach(func() {
-				zoneFileCache = NewZoneFileCache(nameServerIP, domain)
+				zoneFileCache = NewZoneFileCache(nameServerIP, domain, nil)
```

Comment:
I would change at least one on the next tests to use non nil soa number.

Reply:
I'll add a new test for it